pkg/metrics/metrics.go

```diff
@@ -39,6 +39,7 @@ const (
 	backupDeletionAttemptTotal   = "backup_deletion_attempt_total"
 	backupDeletionSuccessTotal   = "backup_deletion_success_total"
 	backupDeletionFailureTotal   = "backup_deletion_failure_total"
+	backupLastSuccessfullSeconds = "backup_last_successful_seconds"
```

Comment:
Any suggestion for a better metric name ?

Reply:
I'd probably go with `backup_last_successful_timestamp` or ``backup_last_successful_timestamp_seconds` - @nrb @carlisia any thoughts? Prometheus docs aren't super-clear about conventions for timestamp metrics.

Reply:
Yeah, I like that name.

Reply:
Me too, `backup_last_successful_timestamp`.